doc/src/build/sui-object-display.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Sui Object Display Standard
+---
+
+The Sui Object Display standard is a template engine that allows for off-chain management of display configurations for a type. With it, you can substitute data for an object into a template string. The standard doesn’t limit the fields you can set. You can use the  `{property}` syntax to access all object properties, and then insert them as a part of the template string.
```

Comment:
Its should be a reverse - on-chain management for off-chain representation (display).